api/src/test/java/org/openmrs/api/AdministrationServiceTest.java

```diff
@@ -449,6 +449,21 @@ public void saveGlobalProperty_shouldFailIfGivenAllowedLocaleListDoesNotContainD
 		adminService.saveGlobalProperty(
 			new GlobalProperty(OpenmrsConstants.GLOBAL_PROPERTY_LOCALE_ALLOWED_LIST, localeList));
 	}
+
+	@Test
+	public void saveGlobalProperty_shouldFailIfDefaultLocaleNotInAllowedLocaleList() {
+
```

Comment:
almost, you did need to change variable names and the assert (although I think we can even remove it)

```java
	@Test
	public void saveGlobalProperty_shouldFailIfGivenDefaultLocaleIsNotInAllowedLocales() {

		Locale defaultLocale = new Locale("fr");

		expectedException.expect(APIException.class);
		expectedException.expectMessage("is not in allowed locales list");

		adminService.saveGlobalProperty(
			new GlobalProperty(OpenmrsConstants.GLOBAL_PROPERTY_DEFAULT_LOCALE, defaultLocale.toString()));
	}
```

Reply:
Done :) 

Thank you @teleivo :) 

What Next?